### PR TITLE
Update tensorflow to 1.2.0

### PR DESCRIPTION
**Pull request by adrianchia**
Fixes https://github.com/conda-forge/tensorflow-feedstock/issues/35

### CODE REVIEW

**Comment by hajapy** [comment]
A few things:

1. https://github.com/tensorflow/tensorflow/issues/11063#issuecomment-311196587 means we can probably use markdown >= 2.6.8
2. The bleach requirement is fine, but you should also add the explicit html5lib ==0.9999999 requirement, otherwise it will use a newer version that may not work
3. Why was windows disabled completely? We should be able to get pip wheels for py35 and py36 now: https://pypi.python.org/pypi/tensorflow
4. Why are there no longer 3 cases to embark on for mac/linux? 
5. This has carried forward for a while now, but the build requirements could be pared down substantially since we just use python, pip and (maybe) wheel to repackage the binaries.

**Comment by adrianchia** [comment]
I believe conda smity rerender caused the removal of 3 environments, let me revert this to previous version along with other changes.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,13 +26,21 @@ requirements:
     - six >=1.10.0
     - wheel
     - werkzeug >=0.11.10
+    - markdown >=2.6.8
+    - bleach ==1.5.0
+    - backports.weakref ==1.0rc1
+    - html5lib ==0.9999999
```
Comment (jakirkham) [inline recipe/meta.yaml]: Admittedly much of this is from before your PR, but could you please drop everything from `build` that isn't strictly required for building the package. Probably can get away with `python`, `pip` only. Then please try re-rendering. Also please make sure you have the latest `conda-smithy` and `conda-build-all`.

Comment (jakirkham) [inline recipe/meta.yaml]: note: This is already done.

**Comment by adrianchia** [comment]
Recipe updated to include Python 3.6 Windows build.

**Comment by hajapy** [comment]
Looks good to me, other changes (small test script & checksum validation) can come in a separate PR

**Comment by hajapy** [comment]
1.2.1 has been released now with the fix to the markdown requirement: https://github.com/tensorflow/tensorflow/releases/tag/v1.2.1

**Comment by jakirkham** [comment]
Went ahead and merged this. Hope that is ok @jjhelmus. Feel free to send an update PR @hajapy.

**Pull request merged**